cli/schema.go

```diff
@@ -0,0 +1,101 @@
+package cli
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"time"
+
+	"github.com/cloudspannerecosystem/harbourbridge/conversion"
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/google/subcommands"
+)
+
+// SchemaCmd struct with flags.
+type SchemaCmd struct {
+	prompt           bool
+	filePrefix       string
+	driverName       string
+	schemaSampleSize int64
+	dumpFilePath     string
+	targetDb         string
+}
+
+// Name returns the name of operation.
+func (cmd *SchemaCmd) Name() string {
+	return "schema"
+}
+
+// Synopsis returns summary of operation.
+func (cmd *SchemaCmd) Synopsis() string {
+	return "generate schema for target db from source db schema"
+}
+
+// Usage returns usage info of the command.
+func (cmd *SchemaCmd) Usage() string {
+	return fmt.Sprintf(`%v schema [file]...
+
+Convert schema for source db specified by driver. Source db dump files can be
+specified by either dump-file flag or from stdin. Connection profile for source
+databases in direct connect mode can be specified by setting appropriate
+environment variables. The schema flags are:
+`, path.Base(os.Args[0]))
+}
+
+// SetFlags sets the flags.
+func (cmd *SchemaCmd) SetFlags(f *flag.FlagSet) {
```

Comment:
Why dont we also add the dbName flag here?

Reply:
We don't want users to have to specify Spanner database info i.e., instance, dbName, (or project_id as part of environment variables). Users will though still have ability to specify `prefix` to name the output files appropriately.